package.json

```
@@ -3,51 +3,10 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e"
+    "postinstall": "./postinstall.sh"
   },
   "private": true,
-  "dependencies": {
-    "@angular/animations": "^5.0.2",
-    "@angular/common": "^5.0.2",
-    "@angular/compiler": "^5.0.2",
-    "@angular/core": "^5.0.2",
-    "@angular/forms": "^5.0.2",
-    "@angular/http": "^5.0.2",
-    "@angular/platform-browser": "^5.0.2",
-    "@angular/platform-browser-dynamic": "^5.0.2",
-    "@angular/router": "^5.0.2",
-    "@ng-bootstrap/ng-bootstrap": "^1.1.2",
-    "classlist.js": "^1.1.20150312",
-    "core-js": "^2.4.1",
-    "rxjs": "^5.4.1",
-    "web-animations-js": "^2.3.1",
-    "zone.js": "^0.8.14"
-  },
   "devDependencies": {
-    "@angular/cli": "^1.5.0",
-    "@angular/compiler-cli": "^5.0.2",
-    "@angular/language-service": "^5.0.2",
-    "@types/jasmine": "~2.5.53",
-    "@types/jasminewd2": "~2.0.2",
-    "@types/node": "~6.0.60",
-    "ces-theme": "https://github.com/cloudogu/ces-theme.git",
-    "codelyzer": "~3.0.1",
-    "jasmine-core": "~2.6.2",
-    "jasmine-spec-reporter": "~4.1.0",
-    "karma": "~1.7.0",
-    "karma-chrome-launcher": "~2.1.1",
-    "karma-cli": "~1.0.1",
-    "karma-coverage-istanbul-reporter": "^1.2.1",
-    "karma-jasmine": "~1.1.0",
-    "karma-jasmine-html-reporter": "^0.2.2",
-    "protractor": "~5.1.2",
-    "ts-node": "~3.0.4",
-    "tslint": "~5.3.2",
-    "typescript": "~2.6.2"
+    "ces-theme": "https://github.com/cloudogu/ces-theme.git"
```

use specific version of the ces-theme e.g.:
```suggestion
    "ces-theme": "https://github.com/cloudogu/ces-theme.git@0.7.2"
```

---

When trying to npm install, the install hangs without any indications as to why, this is also the case with verbose logs. Does it work on your end? I tried "@0.7.2" and "@v0.7.2".